src/IConfigOptions.ts

```diff
@@ -91,6 +91,8 @@ export interface IConfigOptions {
 
     setting_defaults?: Record<string, any>; // <SettingName, Value>
 
+    otp_length?: number;
```

Comment:
Given this OTP varies in length per Matrix homeserver and not per Element instance, this cannot be configurable here

Reply:
Yes, we need OTP length from response of api `_matrix/client/r0/register/msisdn/requestToken`

Reply:
That doesn't seem to be an existing field, this change would need an MSC - https://github.com/matrix-org/matrix-spec-proposals#the-matrix-spec-process